recipes-connectivity/connman/connman/0001-main-add-ve-and-vb-to-the-default-interface-blacklis.patch

```diff
@@ -0,0 +1,32 @@
+From 85c6b7880473e72196203160c4057341c6cad20a Mon Sep 17 00:00:00 2001
+From: Krisztian Litkey <krisztian.litkey@intel.com>
+Date: Thu, 31 Mar 2016 15:14:51 +0300
+Subject: [PATCH 1/1] main: add ve- and vb- to the default interface blacklist.
+
+Add ve- and vb- to the list of default blacklisted interface
+prefixes. Such interfaces are usually used by systemd-nspawned
+containers and are typically managed by systemd-networkd.
+
+Upstream-Status: Pending
+
+Signed-off-by: Krisztian Litkey <krisztian.litkey@intel.com>
+---
+ src/main.c | 2 ++
+ 1 file changed, 2 insertions(+)
+
+diff --git a/src/main.c b/src/main.c
+index e46fa7b..f44a2ed 100644
+--- a/src/main.c
++++ b/src/main.c
+@@ -57,6 +57,8 @@ static char *default_blacklist[] = {
```

Comment:
The purpose of the blacklist patch is to enable the removal of the main.conf. I'm in favor of doing it via a patch, because maintaining the main.conf during connman updates is fragile (continues to use out-dated values without obvious errors - a failure to apply an out-dated patches is more obvious).

I don't find it likely that OEMs will need to recompile ConnMan to add their own default settings. If it is a valid concern, then we are back to the "how can we add better configuration handling to ConnMan" discussion.

My position, as explained in the "ostro-dev: stateless Ostro OS", still is that the core OS should not rely on the ConfFW to populate /etc. It leads to a circular dependency, instead of cleanly layering the ConfFW on top of a stateless OS.

What is currently implemented ("make connman to read main.conf from also from default dir" in https://github.com/pohly/ostro-os/commits/stateless) is that we move /etc/connman/main.conf to /usr. This gets done by the stateless support, there's no need to modify the location of main.conf in the AppFW.

But that does not get around the problem that meta-appfw just isn't a good place for maintaining the distro-specific main.conf. That'll break as soon as any other component will also need to modify the file (ConfFW does not count, it will do that at runtime on top of the main.conf provided currently by AppFW).